react/src/components/organisms/TabContainer/tab.js

```diff
@@ -51,7 +54,8 @@ Tab.propTypes = {
   // When true, the tab will be open by default when used with TabContainer.
   default: PropTypes.bool,
   // The text of the tab.
```

Comment:
can you update this to the comment format storybook reads:
```suggestion
   /** The text of the tab. */
```